internal/cmdrender/cmd.go

```diff
@@ -75,7 +78,8 @@ func (r *Runner) runE(c *cobra.Command, _ []string) error {
 		return err
 	}
 	executor := Executor{
-		PkgPath: r.pkgPath,
+		PkgPath:        r.pkgPath,
+		TruncateOutput: !r.disableOutputTruncate,
```

Comment:
Truncation seems like it is something that is only interesting for printing, so could we put this in the printer instead of passing it down?

Reply:
printer is not a global instance and looks like it's created on demand? How do we set it in printer here?

Reply:
The printer is created in the top-level cobra command: 

https://github.com/GoogleContainerTools/kpt/blob/e0e0b3642969c2d14fe1d38d9698a73f18aa848f/run/run.go#L80

We pass it along as a value attached to the context. Depending on whether truncation of the output is something that cuts across multiple commands (it seems like it might), we can add that flag at the top-level and pass it in when the printer is created. Otherwise, we might want to move the creation of the printer down into the leaf cobra commands.

Reply:
I think `truncation` behavior will probably be applied across all the commands, so lifting it up at the printer level probably makes sense. So the flag needs to be made global and its help needs to be adjusted to make it `kpt fn` agnostic. This will eliminate the need for pushing this flag down the call stack because `printer` is available through the `ctx`.

Reply:
Will do

Reply:
Added a global option